tests/http/server/_base.py

```diff
@@ -335,7 +335,7 @@ def __init__(self, seen_awaits: Set[Tuple[str, ...]], request_number: int):
         self._request_number = request_number
         self._seen_awaits = seen_awaits
 
-        self._original_Deferred___next__ = Deferred.__next__
+        self._original_Deferred___next__ = Deferred.__next__  # type: ignore[misc,unused-ignore]
```

Comment:
What's the misc complaining about OOI?

Reply:
```
tests/http/server/_base.py:338: error: Access to generic instance variables via class is ambiguous  [misc]
```

Reply:
I think it is complaining about snagging a instance level method from a class?